.golangci.yml

```diff
@@ -62,6 +57,12 @@ linters:
   - whitespace
 
 linters-settings:
+  govet:
+    disable:
+      - printf
```

Comment:
Let's instead use this more precise exclude (via exclude-rules below)
```yaml
  # Ignore non-constant format string in call to condition utils
  - linters:
      - govet
    text: "non-constant format string in call to sigs\\.k8s\\.io\\/cluster-api\\/util\\/conditions\\."
```

(+ then fix the one remaining finding)